sources/packages/web/src/composables/institution/useInstitutionState.ts

```diff
@@ -14,8 +14,18 @@ export function useInstitutionState() {
     ).name;
   };
 
+  const initialize = async () => {
+    await store.dispatch("institution/initialize");
+  };
+
+  const setInstitutionUserDetailsOnSetup = async () => {
```

Comment:
I would suggest having it named something like `setInstitutionSetupUser`. It should also just impact/change the `isInstitutionSetupUser` variable, in the way it is right now it is replacing the entire store state.